homeassistant/components/mullvad/binary_sensor.py

```diff
@@ -0,0 +1,42 @@
+"""Setup Mullvad VPN Binary Sensors."""
+from homeassistant.components.binary_sensor import BinarySensorEntity
+
+from . import get_coordinator
+from .const import DOMAIN
+
+BINARY_SENSORS = ("mullvad_exit_ip",)
```

Comment:
I would have expected some kind of mapping to an icon or device class (either of those, never both).

For example, the `mullvad_exit_ip`, indicates if we are connected or not. So the `connectivity` device class would have been fitting for that.

Reply:
Since we only have one BinarySensor and a single device class, i've added the device_class property with a static class of `DEVICE_CLASS_CONNECTIVITY` as suggested.